docs/lib/debugging/fragments/android/dev-menu/setup.md

```diff
@@ -0,0 +1,12 @@
+Run the following commands to add a copy of the file `amplify/.config/local-env-info.json` to the 
+directory `app/src/main/res/raw` in your Android project and rename the copy to `localenvinfo.json`:
```

Comment:
Every file here needs to be updated because the IDE automatically wrapped / created the new lines.

Reply:
Ah, yeah, @eeatonaws we have a convention where we don't wrap paragraphs in this repo. Crazy, but true.